### PR TITLE
Simplify paths for fetching contract classes in tests

### LINKED ISSUE #379: Simplify getting contract paths for testing
When testing with contract classes, it'd be easier to abstract out the contract path and just use the contract name e.g. change `get_contract_class("path/to/Contract.cairo")` into `get_contract_class("Contract")`.

### PR DESCRIPTION
**Pull request by andrew-fleming**
This PR proposes to simplify getting contract classes in the test environment. Instead of passing the entire path as an argument to `get_contract_class`, this PR proposes to pass just the contract name. For example:

```
account_cls = get_contract_class('Account')
erc20_cls = get_contract_class('ERC20')
```

as opposed to:

```
account_cls = get_contract_class('openzeppelin/account/Account.cairo')
erc20_cls = get_contract_class('openzeppelin/token/erc20/ERC20.cairo')
```

---

In the event that multiple preset contracts share the same name, `get_contract_class` includes the optional param `is_path`. The syntax looks like this:

```
account_cls = get_contract_class('openzeppelin/account/Account.cairo', is_path=True)
```

Resolves #379.

### CODE REVIEW

**Review by JulissaDantes — APPROVED** [review]
Looks good!

File: docs/Utilities.md [hunk 1]
```diff
@@ -201,10 +201,16 @@ Memoizing functions allow for quicker and computationally cheaper calculations w
...
 
-A helper method that returns the contract class from the given path. To capture the contract class, simply add the contract path as an argument like this:
+A helper method that returns the contract class from the contract's name. To capture the contract class, simply add the contract's name as an argument like this:
 
 ```python
-contract_class = get_contract_class('path/to/contract.cairo')
+contract_class = get_contract_class('ContractName')
+```
+
+If multiple contracts exist with the same name, then the contract's path must be used instead of the name. To pass the contract's path:
```
Comment (martriay) [inline docs/Utilities.md:210]:
```suggestion
If multiple contracts exist with the same name, then the contract's path must be passed along with the `is_path` flag instead of the name. To pass the contract's path:
```

We should be explicit so users don't miss the flag

File: tests/security/test_initializable.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
 import pytest
 from starkware.starknet.testing.starknet import Starknet
-from utils import TRUE, FALSE, assert_revert, contract_path
+from utils import TRUE, FALSE, assert_revert, get_contract_class
+from signers import MockSigner
 
+signer = MockSigner(12345678987654321)
```
Comment (martriay) [inline tests/security/test_initializable.py:6]: why was this needed?

File: tests/security/test_reentrancy.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
 async def reentrancy_mock():
     starknet = await Starknet.empty()
     contract = await starknet.deploy(
-        "tests/mocks/reentrancy_mock.cairo",
+        contract_class=get_contract_class("reentrancy_mock"),
```
Comment (martriay) [inline tests/security/test_reentrancy.py:14]: how was this working?

File: tests/token/erc721/test_ERC721_Mintable_Burnable.py [hunk 4]
```diff
@@ -32,13 +32,10 @@
...
-    erc721_cls = get_contract_class(
-        'openzeppelin/token/erc721/ERC721_Mintable_Burnable.cairo')
-    erc721_holder_cls = get_contract_class(
-        'openzeppelin/token/erc721/utils/ERC721_Holder.cairo')
-    unsupported_cls = get_contract_class(
-        'tests/mocks/Initializable.cairo')
+    account_cls = get_contract_class('Account')
+    erc721_cls = get_contract_class('ERC721_Mintable_Burnable')
+    erc721_holder_cls = get_contract_class('ERC721_Holder')
+    unsupported_cls = get_contract_class('Initializable')
```
Comment (martriay) [inline tests/token/erc721/test_ERC721_Mintable_Burnable.py:38]: image

**Review by martriay — APPROVED** [review]
Left small comments, great improvement!

File: tests/security/test_reentrancy.py (hunk 3 above)
Comment (andrew-fleming) [reply tests/security/test_reentrancy.py:14]: Without the `contract_class` kwarg, you mean?

File: tests/security/test_initializable.py (hunk 2 above)
Comment (andrew-fleming) [reply tests/security/test_initializable.py:6]: Ah it's not!

File: tests/security/test_reentrancy.py (hunk 3 above)
Comment (martriay) [reply tests/security/test_reentrancy.py:14]: yes
